### DISCUSSION TITLE
Some question about request_reviews.

### DISCUSSION THREAD

**Discussion by jacklightChen** [Mergifyio/mergify#1968] [Q&A] [answered]
Hi, when i use request_reviews of teams, if pull request is sent by a member of the team, mergify does not send request to rest of the team members.

**Comment by jd** [comment]
Hi @jacklightChen!

Could you share a PR URL where you see this (not) happening?

**Comment by 0x4f5da2** [comment, marked as answer]
We tried many times, but it seems that the team members of the teams in the `request_reviewers` configuration did not received any message. We are wondering if there is something wrong with our configuration file or this is an expected behavior.

Here is the url: https://github.com/ccfddl/ccf-deadlines/pull/13

**Reply by jacklightChen** [reply]
@jd Please check this pr and config~

**Reply by jd** [reply]
I think the issue is with the syntax. Can you try setting the team name to just `maintainers`?

**Reply by jd** [reply]
That's interesting. I see the engine does execute the rule and send the request to GitHub, but it seems nothing happens.
Is `maintainers` available for you to ask for a review in the PR if you do it manually on https://github.com/ccfddl/ccf-deadlines/pull/14 ?

**Reply by jacklightChen** [reply]
@jd  oh, thanks for your advice! we notice that we do not make the team accessible to the repo. After we add the authority, it works. Thanks a lot~

**Reply by jd** [reply]
No problem. Unfortunately, GitHub API does not return an error when sending wrong team or users in the review, so there's no way for Mergify to report the problem easily.
